MQTicketService/Dockerfile

```diff
@@ -11,13 +11,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-# Specify base image and platform for the build
-ARG imageName=icr.io/ibm-messaging/mq:latest
-ARG platName=docker.io/library/maven:3.6-openjdk-11
+# Specify default base images 
+ARG baseImageRunStage=icr.io/ibm-messaging/mq:latest
+ARG baseImageBuildStage=docker.io/library/maven:3.6-openjdk-11
```

Comment:
Can this be pre-canned for arm64 builds to save the user the effort of going and working out what is needed. We do this for amd64 already.